src/games/strategy/triplea/Constants.java

```diff
@@ -144,6 +144,7 @@
   String UNIT_TYPE_CARRIER = "carrier";
   String UNIT_TYPE_FIGHTER = "fighter";
   String UNIT_TYPE_BOMBER = "bomber";
+  String UNIT_TYPE_TAC_BOMBER = "tactical_bomber";
```

Comment:
I would have to say that I'm opposed to this idea. Keeping it away from the other unit types would cut across the design and make it harder to maintain. If we shouldn't be using constants,  then what?